InstabugSample/App.js

```diff
@@ -4,7 +4,7 @@
  * @flow
  */
 
-import React, { Component } from 'react';
+import React, { Component } from "react";
```

Comment:
Prettier config file is missing. Please copy it from the dashboard app to InstabugSample folder and reformat the file.
`https://github.com/Instabug/dashboard-app/blob/master/.prettierrc`